client.py

```diff
@@ -16,6 +16,28 @@ def main(args):
         password=args.password,
     )
 
+    # headers = {
+    #     'Host': client.host,
+    #     'Cookie': f'sessionid={client.session_id}; csrftoken={client.csrftoken}',
+    #     'Content-Type': 'application/x-www-form-urlencoded',
+    #     'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,*/*;q=0.8',
+    #     'Accept-Encoding': 'gzip, deflate, compress',
+    #     'Referer': f'https://{client.host}/accounts/login/',
+    #     'Connection': 'close'
+    # }
+    headers = {
+        'Host': client.host,
+        'Cookie': f'sessionid={client.session_id}; csrftoken={client.csrftoken}',
+    }
+
+    flags = []
+    queue = []
+    visited = {}
+    queue.append("/fakebook")
```

Comment:
I think all the hrefs end with `/` - maybe make this `/fakebook/` as well just in case

Reply:
Sounds good